src/test/java/codeu/controller/RegisterServletTest.java

```diff
@@ -61,6 +79,13 @@ public void testDoPost_NewUser() throws IOException, ServletException {
     Mockito.when(mockUserStore.isUserRegistered("test username")).thenReturn(false);
     registerServlet.setUserStore(mockUserStore);
 
+				UUID fakeConversationId = UUID.randomUUID();
+    Conversation fakeConversation =
+        new Conversation(fakeConversationId, UUID.randomUUID(), "test_conversation", Instant.now());
+    Mockito.when(mockConversationStore.getActFeedConversation())
+        .thenReturn(fakeConversation);
+
+
```

Comment:
Just a few comments to clarify some things, but other than that looks good! I really like how you made the activityFeed a special instance of the ConversationStore - it was a smart idea!